src/moving_object.cpp

```diff
@@ -42,3 +42,34 @@ geo::CompositeShapePtr makeWorldSimObject(double width, double length){
 geo::CompositeShapePtr makeWorldSimObject(MovingObject object){
     return makeWorldSimObject(object.width, object.length);
 }
+
+geo::CompositeShapePtr makeApproxRoundWorldSimObject(double radius, int nCorners){
```

Comment:
Do note that geolib also has a function like this for creating cylinders

Reply:
https://github.com/tue-robotics/geolib2/blob/8deca1aae2581bd73cba40a91b55bf3a262d49bd/src/shapes.cpp#L11

Reply:
Due to time limitations, changes to this part are postponed

Reply:
Bad excuses...

Reply:
Also, you should return Shape pointers, as promote them to a CompositeShape pointer only when you need to call CompositeShape only members.